frontend/app/src/App.tsx

```diff
@@ -447,6 +454,17 @@ export class App extends PureComponent<Props, State> {
         this.setLibConfig(libConfig)
       },
     })
+  }
+
+  componentDidMount(): void {
+    // Initialize connection manager here, to avoid
+    // "Can't call setState on a component that is not yet mounted." error.
+    this.initializeConnectionManager()
+
+    this.hostCommunicationMgr.sendMessageToHost({
+      type: "SCRIPT_RUN_STATE_CHANGED",
+      scriptRunState: this.state.scriptRunState,
+    })
```

Comment:
I assume its not problematic that now the `SCRIPT_RUN_STATE_CHANGED` message is sent _before_ the `SET_THEME_CONFIG` message now after the change, but want to sanity-check this change.

Reply:
I believe so -- some manual testing occurred with this happening here, but I'll double check that this should be ok.